vochain/transaction.go

```diff
@@ -230,11 +232,11 @@ func VoteTxCheck(vtx *models.Tx, state *State, txID [32]byte, forCommit bool) (*
 					return nil, fmt.Errorf("cannot digest public key: (%w)", err)
 				}
 
-				// check merkle proof
+				// check census proof
 				var valid bool
-				valid, vp.Weight, err = checkProof(tx.Proof, process.CensusOrigin, process.CensusRoot, vp.PubKeyDigest)
+				valid, vp.Weight, err = checkProof(tx.Proof, process.CensusOrigin, process.CensusRoot, process.ProcessId, vp.PubKeyDigest)
```

Comment:
If I'm not wrong and the processID is not used inside the `checkProof` func the first comment also applies here.

Reply:
Same reply.